apps/scripts/script.txt

```diff
@@ -1,6 +1,6 @@
 GNU_DIR=/home/saar/projects/riscv/install/rv32i/bin/
 mod=i
-$GNU_DIR/riscv32-unknown-elf-gcc -S -ffreestanding -march=rv32$mod Binary_Search.c -o Binary_Search_rv32$mod.c.s;
+$GNU_DIR/riscv32-unknown-elf-gcc -S -ffreestanding -march=rv32$mod $3.c -o $3_rv32$mod.c.s;
```

Comment:
Why do you need this step? Why don't you just build the C file directly into a binary? You're getting the Assembly code anyway with the objdump after the build

Reply:
I guess it's due to a mode of work reference we started with.

we can examine the assembly of our code without any libs.
We will try and run it directly as you suggested.